pkg/platform/runtime/buildexpression/buildexpression.go

```diff
@@ -614,7 +615,7 @@ func (e *BuildExpression) removeRequirement(requirement model.Requirement) error
 	}
 
 	if !found {
-		return errs.New("Could not find requirement")
+		return locale.NewInputError("err_remove_requirement_not_found", "Could not remove requirement with namespace: {{.V0}}, name: {{.V1}}", requirement.Namespace, requirement.Name)
```

Comment:
```suggestion
		return locale.NewInputError("err_remove_requirement_not_found", "Could not remove requirement '[ACTIONABLE]{{.V0}}[/RESET]', because it does not exist.", requirement.Name)
```